src/App.css

```diff
@@ -1,3 +1,17 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
```

Comment:
- [ ] As it is a requirement for this project, please make sure your styling is exactly as the one provided on `Zeplin`. See image  below. 

***The design***
![zaplin](https://user-images.githubusercontent.com/65852621/146470417-f670c7a0-627c-4e36-b2f4-27f043619ae7.png)

***Your implemen
![emmanualI](https://user-images.githubusercontent.com/65852621/146470434-511ca7ba-0661-4283-ac80-6a4c11d6d6b8.png)
tation***

Reply:
Thanks. Fixed